libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SmartStore/SFSmartStoreInspectorViewController.m

```diff
@@ -73,43 +73,41 @@ @interface SFSmartStoreInspectorViewController () <UINavigationBarDelegate>
 
 @implementation SFSmartStoreInspectorViewController
 
-#pragma mark - Singleton
+#pragma mark - Constructor
 
-+ (SFSmartStoreInspectorViewController *) sharedInstance
+- (instancetype) initWithStore:(SFSmartStore*)store
 {
-    static SFSmartStoreInspectorViewController *sharedInstance = nil;
```

Comment:
Not keeping a singleton anymore, but instead exposing a constructor that takes a store.